File: Java/com.faculdadevincit.designpatterns.observerpattern/src/DisplayFarenheitObservable.java
public class DisplayFarenheitObservable implements IObservable<Double>{

    @Override
    public void update(Double value) {
        double farenheit = (value * 9 / 5) + 32;
        System.out.println("Temperatura em Farenheit: " + farenheit + " ºF");
    }

}

File: CSharp/FaculdadeVincit.DesignPatterns.ObserverPattern/DisplayFarenheitObservable.cs
namespace FaculdadeVincit.DesignPatterns.ObserverPattern;

public class DisplayFarenheitObservable : IObservable<double>
{
    public void Update(double valor)
    {
        var fareinheit = (valor * 9 / 5) + 32;
        Console.WriteLine($"Temperatura em Farenheit: {fareinheit} ºF!");
    }

}
